src/common/types/value.cpp

```diff
@@ -244,6 +244,18 @@ Value Value::LIST(vector<Value> values) {
 	return result;
 }
 
+Value Value::BLOB(string data) {
+	Value result(TypeId::VARCHAR);
+	result.sql_type = SQLType::BLOB;
+	result.str_value = data;
+	// pre-allocate to hexadeximal cast in ToString or in CastFromBlob
+	// it should have a double size of the original value plus two for hex identification, i.e., "\x"
+	//! It's need for heap allocation
+	result.hexa_str_value.resize(data.size() * 2 + 2);
```

Comment:
I don't quite see why the "hexa_str_value" variable is necessary in the value, this could be removed no?